src/frontend/src/components/AuthorResult/AuthorResult.jsx

```diff
@@ -55,12 +51,13 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-export default function AuthorResult() {
+export default function AuthorResult(props) {
   const classes = useStyles();
-  const [open, setOpen] = React.useState(false);
+  const { author } = props;
+  const { postDate, title, postLink } = props.post;
 
-  const handleClick = () => {
-    setOpen(!open);
+  const handleLatestPostClick = () => {
+    window.open(postLink);
```

Comment:
This may be problematic.  Gatsby doesn't have a `window` at build time.  Have you tried doing a full build of your code?  Also, why are we opening a new window?

Reply:
I think opening a new window when the user clicks on a post makes sense. I'm gonna need to ask you more about `window` for the PWA thing on a later date